### PR TITLE
TINKERPOP-1654 Bumped to Jackson 2.8.7

### PR DESCRIPTION
**Pull request by spmallette**
https://issues.apache.org/jira/browse/TINKERPOP-1654

Did a build with `mvn clean install && mvn verify -pl gremlin-server -DskipIntegrationTests=false` but expect to do a full build with docker for all branches this merges too after vote. 

VOTE +1

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -26,6 +26,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 TinkerPop 3.1.7 (Release Date: NOT OFFICIALLY RELEASED YET)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* Bumped to Jackson 2.7.9.
```
Comment (robertdale) [inline CHANGELOG.asciidoc]: Update version to 2.8.7

Comment (spmallette) [inline CHANGELOG.asciidoc]: dah - no way....

rebased with a fixup. shoudl be good now.

**Comment by pluradj** [comment]
Did a build with `mvn clean install && mvn verify -pl gremlin-server -DskipIntegrationTests=false` and Docker build with tests on `tp32`. Looks fine.

VOTE: +1

**Comment by robertdale** [comment]
VOTE +1

**Pull request merged**